docs/UserGuide.md

```diff
@@ -399,9 +400,19 @@ If your changes to the data file makes its format invalid, UGTeach will discard
 Furthermore, certain edits can cause the application to behave in unexpected ways (e.g., if a value entered is outside the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
 </box>
 
+
 --------------------------------------------------------------------------------------------------------------------
+
 <div style="page-break-after: always;"></div>
 
+## Glossary
+
+* **Duplicated student:** Two students are duplicates if they have the exact same **NAME** (case-sensitive) and the same **PHONE NUMBER** at the same time.
```

Comment:
![image](https://github.com/user-attachments/assets/8862eb73-8f99-46ce-9edb-ceeb38138eaf)

I am not sure what u mean by exact same **NAME** (case-sensitive). I tried to add another john with lowercase letters and it is not acceptable

Maybe "Two students are duplicates if they have the same **NAME** and the same **PHONE NUMBER**?

Reply:
I see what you mean